src/SearchInput/SearchInput.js

```diff
@@ -34,11 +34,13 @@ export class SearchInput extends Component {
             value: event.target.value
         });
         if (this.props.onChange) {
-            this.props.onChange();
+            this.props.onChange(event);
         } else {
             if (this.state.searchList) {
                 let filteredResult = this.state.searchList.filter(item =>
-                    item.text.toLowerCase().startsWith(event.target.value.toLowerCase())
+                    item.text
```

Comment:
@aerfio - adding the event argument to the this.props.onChange method looks good. But there seems to be some autoformatting applied to the file. Can you either remove the auto-formatting or if you do have a formatter installed in the IDE have it apply the eslint rules that are available in the repo (.eslintrc)?

Reply:
Doesn't it do it automatically on before-commit hook? I've even run `npm run build:lint:fix` and `npm run lint`

Reply:
OK, my prettier config somehow took precedence before eslint, my bad